spec/routing/address_routing.rb

```diff
@@ -1,62 +0,0 @@
-# frozen_string_literal: true
```

Comment:
This spec was testing our conditional routing logic that no longer exists - which is why we don't need it anymore.